src/v2/util/UserAgentUtil.js

```diff
@@ -0,0 +1,9 @@
+import DeviceFingerprint from '../../util/DeviceFingerprint';
+
+const isAndroid = function () {
```

Comment:
Might as well add this to DeviceFingerprint

Reply:
Or just move that `getUserAgent` method into this file.

Reply:
I prefer to add it to `DeviceFingerprint` since `getUserAgent` is necessary in `DeviceFingerprint` and moving it is quite a big change. 
But adding a simple android user agent check involves much smaller changes. I think we don't even need to add test to `DeviceFingerprint_spec.js`
@nikhilvenkatraman-okta @jhe-okta